src/fcast.c

```diff
@@ -96,92 +100,3 @@ SEXP fcast(SEXP lhs, SEXP val, SEXP nrowArg, SEXP ncolArg, SEXP idxArg, SEXP fil
   UNPROTECT(1);
   return(ans);
 }
-
```

Comment:
why to remove those lines?

Reply:
It is confusing to keep lots of commented code in main source. If somebody wants this code they can always go back and see them in git history.